packages/contracts-bedrock/scripts/deploy/verify-new-chain.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
+
+# Require env vars
+[ -z "${NETWORK:-}" ] && echo "Need to set the NETWORK via env" && exit 1;
```

Comment:
We should create notion document that will state what needs to be rebased and this should be part of it

Reply:
I'm moving this one to Monorepo: https://github.com/celo-org/celo-monorepo/pull/11470
Resolving 😁